README.md

```diff
@@ -49,7 +49,7 @@ Please submit bugs and feature requests in [GitHub Issues](https://github.com/Mi
 
 ## Building for Windows
 
-To build the `.Net Framework 4.5.1` version, you can either built the 
+Make sure Visual Studio 2015 is installed. To build the `.Net Framework 4.5.1` version, you can either built the 
```

Comment:
Visual Studio is not mandatory for build.bat to work. Installing [Visual C++ 2015 Build Tools](http://landinghub.visualstudio.com/visual-cpp-build-tools) can also get MSBuild and VC++ cimplier.